### PR TITLE
Feature/164/create register page

### PR DESCRIPTION
**Pull request by OlkaA**
Created register page

- changed webpack to allow work with jpg format

- added migration to add field user_name in table users

- added condition in controller for users, to not allow write data to DB if user already exists

- authenticate with google, twitter, facebook, email

For testing facebook account ask @OlkaA  or @Laila1222 for adding user to facebook app.

There is still a problem with API key. 
I had to hard code API key and authDomain to make it work.

### CODE REVIEW

**Review by laila1222 — APPROVED** [review]
Works perfectly! Great job! 🥇

File: src/client/pages/Register/RegisterPage.js [hunk 1]
```diff
@@ -1,7 +1,144 @@
...
+
+  onInputChange = ({ name, userName, email, password }) => {
+    this.setState({ name, userName, email, password });
+  };
+
+  render() {
+    return (
+      <Consumer>
+        {() => {
+          return (
```
Comment (bytewiz) [inline src/client/pages/Register/RegisterPage.js]: What is the `<Consumer>` used for here?

**Review by bytewiz — APPROVED** [review]
Looks good to me 👌 Just one comment

File: src/client/pages/Register/RegisterPage.js (hunk 1 above)
Comment (OlkaA) [reply src/client/pages/Register/RegisterPage.js]: Probably we don't need it here.

**Pull request merged**